src/utils/validators.utils.js

```diff
@@ -76,14 +78,36 @@ export const isSkipEntry = (entryMap) => {
   }
 };
 
+/**
+ * Validates the GitHub App installation status in the forked repository.
+ * If the GitHub App is not installed or is suspended, the function returns false.
+ * Otherwise, it returns true.
+ * @param {Object} prData - An object containing the pull request data.
+ * @returns {Boolean} - True if the GitHub App is installed and not suspended, false otherwise.
+ */
+
+export const isGitHubAppInstalledOrNotSuspended = async (prData) => {
+  const { data: githubAppInstallationInfo } =
+    await forkedAuthServices.getGitHubAppInstallationInfoFromForkedRepo(
+      prData.headOwner,
+      prData.headRepo
+    );
+  if (
+    githubAppInstallationInfo.installed ||
+    githubAppInstallationInfo.suspended === false
```

Comment:
```suggestion
    githubAppInstallationInfo.installed &&
    githubAppInstallationInfo.suspended === false
```

Shouldnt it be this? If so even the name of the function needs to be updated

Reply:
You right! My bad there! I will update the code.